src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -154,8 +154,27 @@ public HyperwalletUser updateUser(HyperwalletUser user) {
      * @param options List filter option
      * @return HyperwalletList of HyperwalletUser
      */
-    public HyperwalletList<HyperwalletUser> listUsers(HyperwalletPaginationOptions options) {
+    public HyperwalletList<HyperwalletUser> listUsers(HyperwalletUsersListPaginationOptions options) {
         String url = paginate(this.url + "/users", options);
+        if (options != null) {
+            if (options.getClientUserId() != null) {
```

Comment:
The addParameter function already checks for null values. So just check whether the options is null or not and then call the addParameter without null checks. 
This is a generic comment. Please change in all the places.

Reply:
Done for the whole file

Reply:
Made changes for the whole file.